lib/index.js

```diff
@@ -17,6 +15,10 @@ export {routeGuards} from './router.js';
 // load bedrock-vue-wallet config
 import './config.js';
 
+// Satisfies vue console warning
```

Comment:
What is the vue warning this negates?

Reply:
Vue generates this console warning when those values are not specified

![warnings](https://github.com/digitalbazaar/bedrock-vue-wallet/assets/56396286/a224272a-ae02-42f5-8909-618f8f103c69)

Reply:
We might want the PROD_DEVTOOLS are some point, I get the feeling this is supposed to be set somewhere else too.

Reply:
I removed this change per @dlongley 's comment at the bottom. I will create an issue to address this so that we can more appropriately address this vue warning in the future. See [481125d](https://github.com/digitalbazaar/bedrock-vue-wallet/pull/62/commits/481125d4e3285f5a662da99c1e5832067be4efb1)

Reply:
_New Issue: #64_